packages/login/src/modules/login/sagas.js

```diff
@@ -0,0 +1,17 @@
+import * as actions from './actions'
+import {Pages} from '../../types/Pages'
+
+import {fork, put, takeLatest, all} from 'redux-saga/effects'
+
+export function* usernameLoginSaga({payload}) {
+  if (payload.username) {
+    yield put(actions.setUsername(payload.username))
+    yield put(actions.changePage(Pages.LOGIN_FORM))
```

Comment:
why only change page to login if the username is set? Is this line even needed at all?